### ISSUE TITLE
Release is still pointing dependencies to Kotlin JRE

### ISSUE THREAD

**Issue by Hydragyrum** [authzee/kotlin-guice#5]
Hello, 

Just to let you know that the released version on maven central (the one being pulled by gradle) is still pointing its kotlin dependencies to `org.jetbrains.kotlin:kotlin-stdlib-jre8:1.1.4-3`

Pull Request #4 fixes this on master, but the release was never officially made it seems. 

This is causing a few jar clashes between Kotlin versions locally.

Cheers

**Comment by johnlcox** [comment]
1.1.0 has been released and is in maven central.

**Comment by Hydragyrum** [comment]
Thank-you kindly

**Comment by Hydragyrum** [comment]
Something didn't upload right, as the POM is still pointing to the JRE dependency in the dependencies section, though under DependencyManagement it seems to be alright.

from [Maven Central](https://search.maven.org/artifact/com.authzee.kotlinguice4/kotlin-guice/1.1.0/jar)
```xml
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <modelVersion>4.0.0</modelVersion>
  <groupId>com.authzee.kotlinguice4</groupId>
  <artifactId>kotlin-guice</artifactId>
  <version>1.1.0</version>
  <name>Guice DSL extensions for Kotlin</name>
  <description>Guice DSL extensions for Kotlin</description>
  <url>https://github.com/authzee/kotlin-guice/</url>
  <licenses>
    <license>
      <name>The Apache Software License, Version 2.0</name>
      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
  <developers>
    <developer>
      <id>johnlcox</id>
      <name>John Leacox</name>
    </developer>
  </developers>
  <scm>
    <connection>scm:git:git://github.com/authzee/kotlin-guice.git</connection>
    <developerConnection>scm:git:ssh://git@github.com/authzee/kotlin-guice.git</developerConnection>
    <url>https://github.com/authzee/kotlin-guice/</url>
  </scm>
  <dependencies>
    <dependency>
      <groupId>com.google.inject</groupId>
      <artifactId>guice</artifactId>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>org.jetbrains.kotlin</groupId>
      <artifactId>kotlin-stdlib-jre8</artifactId>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>org.jetbrains.kotlin</groupId>
      <artifactId>kotlin-reflect</artifactId>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>com.google.inject.extensions</groupId>
      <artifactId>guice-multibindings</artifactId>
      <scope>test</scope>
    </dependency>
    <dependency>
      <groupId>org.amshove.kluent</groupId>
      <artifactId>kluent</artifactId>
      <scope>test</scope>
    </dependency>
    <dependency>
      <groupId>org.jetbrains.spek</groupId>
      <artifactId>spek-api</artifactId>
      <scope>test</scope>
    </dependency>
    <dependency>
      <groupId>org.jetbrains.spek</groupId>
      <artifactId>spek-junit-platform-engine</artifactId>
      <scope>test</scope>
    </dependency>
    <dependency>
      <groupId>org.junit.platform</groupId>
      <artifactId>junit-platform-runner</artifactId>
      <scope>test</scope>
    </dependency>
  </dependencies>
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>org.jetbrains.spek</groupId>
        <artifactId>spek-api</artifactId>
        <version>1.1.4</version>
      </dependency>
      <dependency>
        <groupId>com.google.inject</groupId>
        <artifactId>guice</artifactId>
        <version>4.1.0</version>
      </dependency>
      <dependency>
        <groupId>org.jetbrains.kotlin</groupId>
        <artifactId>kotlin-stdlib-jdk8</artifactId>
        <version>1.1.4-3</version>
      </dependency>
      <dependency>
        <groupId>org.amshove.kluent</groupId>
        <artifactId>kluent</artifactId>
        <version>1.28</version>
      </dependency>
      <dependency>
        <groupId>org.jetbrains.kotlin</groupId>
        <artifactId>kotlin-reflect</artifactId>
        <version>1.1.4-3</version>
      </dependency>
      <dependency>
        <groupId>com.google.inject.extensions</groupId>
        <artifactId>guice-multibindings</artifactId>
        <version>4.1.0</version>
      </dependency>
      <dependency>
        <groupId>org.jetbrains.spek</groupId>
        <artifactId>spek-junit-platform-engine</artifactId>
        <version>1.1.4</version>
      </dependency>
      <dependency>
        <groupId>org.junit.platform</groupId>
        <artifactId>junit-platform-runner</artifactId>
        <version>1.0.0-RC3</version>
      </dependency>
    </dependencies>
  </dependencyManagement>
</project>
```

**Comment by johnlcox** [comment]
I think I see the problem.

The [build.gradle](https://github.com/authzee/kotlin-guice/blob/dce267dc3a0e8332d94469ab3202f41c27a37e19/kotlin-guice/build.gradle#L3) file is still referencing it. I'll try to get that fixed tonight.

**Comment by johnlcox** [comment]
I released 1.2.0 that should actually fix this.

**Comment by Hydragyrum** [comment]
Looks good, warnings are gone. Thanks for the quick reaction!
